pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -669,6 +669,7 @@ form {
   background: var(--neutral_light);
   border: 1px solid var(--neutral_dark20);
   border-radius: $border-radius;
+  text-align: initial;
```

Comment:
Added this to the individual action block tiles to keep the text-aligned left (or right for RTL) if a section becomes centered.